shared/interface/Pvfs/ILinearPvf.ts

```diff
@@ -0,0 +1,7 @@
+import {TPvfDirection} from '@shared/types/TPvfDirection';
+import IPvf from '../Problem/IPvf';
+
+export interface ILinearPvf extends IPvf {
+  type: 'linear';
+  direction: TPvfDirection;
```

Comment:
it's kind of weird to have both interfaces that extend `IPvf` both have an identical `direction` property, indicating their base shares that as well